src/options.rs

```diff
@@ -5,4 +5,7 @@ use structopt::StructOpt;
 pub struct Options {
     #[structopt(name = "INPUT", help = "Path to the input file.")]
     pub path: String,
+
+    #[structopt(long = "--overflow-check", help = "Execution panics on integer overflow")]
+    pub overflow_check: bool
```

Comment:
Is this option optional? Does structopt have a notion of a default value for this case then? Should our default be `false`?

Reply:
if the flag is not passed, `overflow_check` is `false`. Is that what you mean?

Reply:
Yes it just wasn't clear from the documentation and for such things I think being explicit is better.

Reply:
sure, I understand.